public/canjs/controls/select-issue-type/select-issue-type.js

```diff
@@ -111,7 +103,7 @@ export class SelectIssueType extends StacheElement {
         let dropdown = new TypeSelectionDropdown().initialize({
             primaryIssueType: this.routeData.primaryIssueType,
             secondaryIssueType: this.routeData.secondaryIssueType,
-            issueHierarchy: this.issueHierarchy,
+            issueHierarchy: this.routeData.issueHierarchy,
```

Comment:
can we make `routeData` available on `TypeSelectionDropDown`? Thenn we don't need to pass anything.

Reply:
Yes, and it's now done.